crates/turborepo-lib/src/run/summary/task.rs

```diff
@@ -251,12 +251,10 @@ impl From<SharedTaskSummary<TaskId<'static>>> for SharedTaskSummary<String> {
             dependencies: dependencies
                 .into_iter()
                 .map(|task_id| task_id.task().to_string())
-                .sorted()
```

Comment:
This `from` implementation is used by the `into()` call here: https://github.com/vercel/turbo/blob/1dd2e93e84e08872f67943804a6600f0e91a124a/crates/turborepo-lib/src/run/summary/task.rs#L211

it only runs for `SinglePackageTaskSummary`. Since we have moved the sorting to the `normalize()` step, we shouldn't need it here.